packages/web-app-admin-settings/src/components/Users/SideBar/DetailsPanel.vue

```diff
@@ -34,6 +34,21 @@
             <span v-if="user.appRoleAssignments" v-text="roleDisplayName" />
           </td>
         </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="$gettext('Quota')" />
+          <td>
+            <span v-if="showUserQuota" v-text="quotaDisplayValue" />
+            <span v-else class="oc-text-meta" v-translate>
```

Comment:
@JammingBen wrote somewhere that the way to go for translations is now to use `v-text="$gettext(...)"` and not use the `v-translate` directive anymore. I don't remember why though...

Reply:
Ok

Reply:
Yes that's right, because it's deprecated (same as `<translate>...</translate>`).